packages/node_modules/@ciscospark/react-container-read-receipts/src/index.js

```diff
@@ -37,13 +37,12 @@ export class ReadReceipts extends Component {
   render() {
     const {users, totalCount} = this.props.readReceipts;
     const readReceipts = users.map((user) =>
-      <TypingAvatar
+      (<TypingAvatar
```

Comment:
What does wrapping in parens give us?

Reply:
This was enabled https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md. Gives better readability for multi-line jsx

Reply:
![img](https://media1.giphy.com/media/26xBHgyz36zV0BOfu/giphy.gif?cid=e1bb72ff5a26c6026136317a6be2ed9b)

No more eslint disable!